chains/runtime-common/src/universal_aliases.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+extern crate alloc;
+
+use {
+    alloc::collections::btree_set::BTreeSet,
+    frame_support::{parameter_types, traits::Contains},
+    xcm::latest::prelude::*,
+};
+
+/// The pallet index of the Ethereum inbound queue pallet in the Tanssi runtime.
+pub const PARENT_INBOUND_QUEUE_PALLET_INDEX: u8 = 24;
+
+parameter_types! {
+    /// Network and location for the Ethereum chain. On Starlight, the Ethereum chain bridged
+    /// to is the Ethereum mainnet, with chain ID 1.
+    /// <https://chainlist.org/chain/1>
+    /// <https://ethereum.org/en/developers/docs/apis/json-rpc/#net_version>
+    pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 11155111 };
```

Comment:
this is actually a bit missleading. I would prefer if we clearly state that this is the Sepolia network id. before I guess this was not in common so it really did not matter, but now being in common, I think it is important